src/app/layout/components/navigation-menu/navigation-menu.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="nav-menu-content">
+  <mat-nav-list>
+    <div class="sidenav-link">
+      <a mat-list-item routerLink='/timeSheet' (click)='onLinkClick()'>
```

Comment:
do we need this click handler?

Reply:
It was used for closing navigation menu on click, but after consultation with Lejla, we decided that we don't need that functionality.